rac_schemas/schemas/rights_copyright.json

```diff
@@ -0,0 +1,49 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "rights_copyright.json",
+	"type": "object",
+	"title": "RAC Copyright Information Schema",
+	"required": [
+			"copyright_status",
+			"copyright_jurisdiction"
+	],
+	"properties": {
+		"copyright_identifier": {
+			"$id": "#/properties/copyright_identifier",
+			"type": "string",
+			"title": "Copyright Documentation Identifier",
+			"minLength": 1
+		},
+		"copyright_status": {
+			"$id": "#/properties/copyright_status",
+			"type": "string",
+			"title": "Copyright Status",
+			"enum": [
+				"copyrighted",
+				"public domain",
+				"unknown"
+			],
+			"examples": [
+				"public domain"
+			]
+		},
+		"copyright_jurisdiction": {
+			"$id": "#/properties/copyright_jurisdiction",
+			"type": "string",
+			"title": "Copyright Jurisdiction",
+			"minLength": 1,
+			"examples": [
+				"FR"
+			],
+			"pattern": "^[A-Z]{2}$"
```

Comment:
I'm pretty sure all our systems use lower case for country codes--does that sound right @helrond ?

Reply:
Well, [this](https://docs.rockarch.org/premis-rights-guidelines/guidelines/#controlled-vocabularies-and-standards) says use ISO 3166, which is uppercase...so if we're using lowercase elsewhere we may be doing it wrong?

Reply:
Okay--that piece of the local guidelines was definitely modeled off of the Archivematica documentation, which calls for that standard but uses lowercase: https://www.archivematica.org/en/docs/archivematica-1.12/user-manual/metadata/premis/#copyright-jurisdiction 🤷‍♀️

It looks like the existing fixtures we have (like in Ursa Major) are all lowercase, so should we continue with that convention and update the docs?

Reply:
Good times.

OK, it looks like Aurora also produces lowercase as well. So yes, let's update docs to say lowercase and enforce that in this schema.